client/benches/tps/utils.rs

```diff
@@ -222,15 +222,15 @@ impl MeasurerUnit {
         let submitter = self.client.clone();
         let interval_us_per_tx = self.config.interval_us_per_tx;
         let instructions = self.instructions();
-        let alice_id = <Account as Identifiable>::Id::from_str("alice@wonderland")
-            .expect("Failed to parse account id");
+        let account_id = account_id(self.name);
```

Comment:
Postfix notation? 
```rust
self.name.account_id()
```

Reply:
this is actually calling a function in this module, not a method on `name`